pkg/apis/mysql/v1alpha1/types.go

```diff
@@ -74,6 +74,7 @@ type ClusterSpec struct {
 	BackupSchedule   string `json:"backupSchedule,omitempty"`
 	BackupUri        string `json:"backupUri,omitempty"`
 	BackupSecretName string `json:"backupSecretName,omitempty"`
+	BackupScheduleJobsHistoryLimit *int `json:"backupScheduleJobsHistoryLimit"`
```

Comment:
Please rebase on current master, since it checks for go formatting issues on CI.